src/query.c

```diff
@@ -498,6 +498,35 @@ IndexIterator *Query_EvalTokenNode(QueryEvalCtx *q, QueryNode *qn) {
   return NewReadIterator(ir);
 }
 
+static void addTerm(char *str, size_t tok_len, QueryEvalCtx *q, IndexIterator ***its, size_t *itsSz, size_t *itsCap, QueryNodeOptions *opts) {
```

Comment:
consider making addTerm inline.

Also consider changing arguments to:
`(RSToken* tok, QueryEvalCtx *q, IndexIterator ***its, size_t *itsSz, size_t *itsCap, QueryNodeOptions *opts) {`

Reply:
I'm passing a string so function arguments are different.
Made it inline